standard/lexical-structure.md

```diff
@@ -52,7 +52,18 @@ Every compilation unit in a C# program shall conform to the *compilation_unit*
 
 ### 6.2.5 Grammar ambiguities
 
-The productions for *simple_name* ([§12.8.4](expressions.md#1284-simple-names)) and *member_access* ([§12.8.7](expressions.md#1287-member-access)) can give rise to ambiguities in the grammar for expressions.
+The productions for:
+
+- *simple_name* ([§12.8.4](expressions.md#1284-simple-names)),
+- *member_access* ([§12.8.7](expressions.md#1287-member-access)),
+- *null_conditional_member_access* ([§12.8.8](expressions.md#1288-null-conditional-member-access)),
+- *dependent_access* ([§12.8.8](expressions.md#1288-null-conditional-member-access)),
+- *base_access* ([§12.8.15](expressions.md#12815-base-access)) and
+- *pointer_member_access* ([§23.6.3](unsafe-code.md#2363-pointer-member-access));
+
+(the “disambiguated productions”) can give rise to ambiguities in the grammar for expressions.
```

Comment:
```suggestion
can give rise to ambiguities in the grammar for expressions. Within this clause, these are known as *disambiguated productions*.
```

(Just as one possible option. I'm not really that bothered.)

Reply:
I'm fine either way.

Reply:
I didn’t use *italics* but quoted it as we use those for proper defined terms, the rest is just the style I’m used to, i.e. a shorthand in paren. But I fine either way as well. @RexJaeschke what would you do here?

Reply:
I'm happy enough to leave this.